docs/gen-ai/gen-ai-metrics.md

```diff
@@ -2,28 +2,32 @@
 linkTitle: Generative AI metrics
 --->
 
-# Semantic Conventions for Generative AI Client Metrics
+# Semantic Conventions for Generative AI Metrics
 
 **Status**: [Experimental][DocumentStatus]
 
-The conventions described in this section are specific to Generative AI client
-applications.
-
-**Disclaimer:** These are initial Generative AI client metric instruments
-and attributes but more may be added in the future.
-
 <!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
 
 <!-- toc -->
 
 - [Generative AI Client Metrics](#generative-ai-client-metrics)
   - [Metric: `gen_ai.client.token.usage`](#metric-gen_aiclienttokenusage)
   - [Metric: `gen_ai.client.operation.duration`](#metric-gen_aiclientoperationduration)
+- [Generative AI Model Server Metrics](#generative-ai-model-server-metrics)
+  - [Metric: `gen_ai.server.latency.request_duration`](#metric-gen_aiserverlatencyrequest_duration)
+  - [Metric: `gen_ai.server.latency.time_per_output_token`](#metric-gen_aiserverlatencytime_per_output_token)
```

Comment:
similarly to `gen_ai.server.time_to_first_token`, why not `gen_ai.server.time_per_token`? 
Also if do `time_per_output_token`, we should probably do `time_to_first_output_token`

Reply:
Time to first token (TTFT) and time per output token (TPOT) are widely used terminologies as you can see from the best practices guides and usage across companies and organizations - [Databricks](https://www.databricks.com/blog/llm-inference-performance-engineering-best-practices), [Microsoft](https://learn.microsoft.com/en-us/azure/databricks/machine-learning/foundation-models/prov-throughput-run-benchmark#--llm-inference-introduction), [ MLCommons ](https://mlcommons.org/2024/03/mlperf-llama2-70b/) and [vLLM](https://github.com/vllm-project/vllm/blob/828da0d44e9124d949909477d6018fc08469a31e/vllm/engine/metrics.py#L77). So, I'd like to keep them as is.